### ISSUE TITLE
[Bug Report] Cover Images Missing After Updates

### ISSUE THREAD

**Issue by MrSoadDicted** [stashapp/stash#4405]
**Describe the bug**
Stash keeps losing cover images/screenshots after updates to Stash Docker container.

I've finally figured out that the issue happens each time after the Stash Docker container receives an update. It may even happen after restarting the container as well, but for sure after an update.

For the longest time only the newest videos in Stash had cover images and after the second to last most recent update (there were two updates in about as many days), the rest of the cover images went missing. I finally went through with regenerating everything by checking the Overwrite Existing Files toggle in settings, then clicking Generate. That took a little more than a full day to complete, but everything had cover images. Then the most recent update released so I updated the Stash Docker container again, and now everything is without cover images once again. 

I searched the Issues already submitted and can't find anyone else submitting this particular issue so here I am.

**To Reproduce**
1. Rescan entire library with "Overwrite Existing Files" toggled under Generated Content.
2. Wait for completion, at which point all videos should now have cover images.
3. Update Stash Docker container (possibly even just restarting the container will do it)
4. All cover images are now missing

**Expected behavior**
For Stash to retain cover images for all videos after an update/restart of the Docker Container.

**Screenshots**
[Screenshot showing the missing cover images](https://i.imgur.com/Lpr9N9M.png)

**Stash Version: (from Settings -> About):**
v0.24.1 (has been happening for as long as I've used Stash, probably close to a year, so throughout multiple versions)

**Desktop (please complete the following information):**
- Running as a Docker container on Unraid 6.11.5
- Official image (stashapp/stash)

**Smartphone (please complete the following information):**
 N/A - The issue is visible on all client platforms

**Additional context**
N/A

**Comment by scruffynerf** [comment]
come on, @MrSoadDicted you can't even change the title to something meaningful?

**Comment by bob123491234** [comment]
It sounds like you haven't added a docker volume mount for the blobs directory, add that and the cover images should persist through updates.

**Comment by kyuu840** [comment]
On the Unraid Docker page, edit your Stash container, and set the _Blobs Path_ variable to a path where they will be saved. For example, `/mnt/user/appdata/stash/blobs/`. If you don't do that, the blobs will be saved inside the docker container, and deleted every time the container is restarted.

**Comment by MrSoadDicted** [comment]
Genuine apologies for the missing title. I proofread my post several times and somehow never noticed. I've corrected the title to something useful.

@bob123491234 and @kyuu840 I think you nailed it. I checked the Docker template and the blobs path was missing completely. It looks like it wasn't included by the template maintainer (in Unraid's Community Applications) when I first installed it however long ago, though I checked the template in CA and it has since been updated to include that path.

I added it to my template and I'm re-generating and the blobs directory appears to be slowly populating so I think this is solved. I'll follow up after testing it if not. 

Thanks for the help and I apologize again for the title issue.
